tests/python/test_simt.py

```diff
@@ -275,8 +275,29 @@ def test_match_any():
 
 @test_utils.test(arch=ti.cuda)
 def test_match_all():
-    # TODO
-    pass
+    a = ti.field(dtype=ti.i32, shape=32)
+    b = ti.field(dtype=ti.u32, shape=32)
+    c = ti.field(dtype=ti.u32, shape=32)
+
+    @ti.kernel
+    def foo():
+        ti.loop_config(block_dim=32)
+        for i in range(32):
+            a[i] = 0
```

Comment:
can we change this value to a non-zero? because fields are already initialized with zeros.

Reply:
Sure!